beacon_api/api/info.py

```diff
@@ -21,6 +21,7 @@
     :return beacon_info: A dict that contain information about the ``Beacon`` endpoint.
     """
     beacon_dataset = await fetch_dataset_metadata(pool)
+    beaconhandover = make_handover(__handover_beacon__, [x['id'] for x in beacon_dataset])
```

Comment:
`beaconHandover` and `datasetHandover` are optional properties, and shouldn't be always added.

Handover field should be added when:
* Handover drs is configured (meaning handover feature is enabled).

Handover field should not be added when:
* Handover drs is not configured (meaning handover feature is disabled).